Dockerfile.web

```diff
@@ -0,0 +1,21 @@
+#DOCKERFILE SPECIFICIALLY FOR HEROKU AND USING A NONROOT USER FOR INSTALLATION
+#ADDING IN THE ABILITY TO ENSURE A SUCCESSFUL LAUNCH WITHOUT ROOT
```

Comment:
It looks like you've updated your production target in the main Dockerfile to also handle the non-root case - is there any reason to keep (& use) this file?